stdlib/2and3/mmap.pyi

```diff
@@ -71,6 +70,9 @@ if sys.version_info >= (3,):
         def __setitem__(self, index: int, object: int) -> None: ...
         @overload
         def __setitem__(self, index: slice, object: bytes) -> None: ...
+        # Doesn't actually exist, but the object is actually iterable because it has __getitem__ and
+        # __len__, so we claim that there is also an __iter__ to help type checkers.
```

Comment:
Yeah, the CompatInterable solution doesn't really seem to work unless we start using it everywhere. I am fine putting this off though, and the solution for mmap() feels acceptable.